### PR TITLE
Add SwButton

### PR DESCRIPTION
**Pull request by tomecko**

### CODE REVIEW

File: src/input/button/button.sass [hunk 1]
```diff
@@ -0,0 +1,7 @@
+@import "~bourbon/core/bourbon"
+
+.sw-button
+  position: relative
+
+  &__spinner
+    +position(null, null -20px null auto)
```
Comment (ulfryk) [inline src/input/button/button.sass]: shouldn't spinner position depend on `bsSize` ?

File: src/input/button/button.sass [hunk 2]
```diff
@@ -0,0 +1,7 @@
+@import "~bourbon/core/bourbon"
+
+.sw-button
+  position: relative
```
Comment (ulfryk) [inline src/input/button/button.sass]: shouldn't button hold additional padding for spinner ?

File: src/input/button/button.tsx [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+  return (
+    <Button
+      {...props}
+      className={cx(props.className, css.COMPONENT)}
+      disabled={pending || props.pending}
+    >
+      {props.children}
+      <SwSpinner
+        className={css.SPINNER}
+        config={smallSpinner}
```
Comment (ulfryk) [inline src/input/button/button.tsx]: shouldn't spinner size depend on `bsSize` ? at least for `xs`

File: src/input/button/button.tsx [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+
+import './button.sass';
+
+function SwButton(props: ISwButtonProps) {
+  const { pending } = props;
+  return (
+    <Button
+      {...props}
+      className={cx(props.className, css.COMPONENT)}
+      disabled={pending || props.pending}
```
Comment (ulfryk) [inline src/input/button/button.tsx]: `true || true` or `false || false` ;)

File: src/input/button/button.tsx [hunk 5]
```diff
@@ -0,0 +1,33 @@
...
+import * as css from './class-names';
+import { ISwButtonProps, SW_BUTTON_DEFAULT_PROPS, SW_BUTTON_PROP_TYPES } from './config';
+
+import './button.sass';
+
+function SwButton(props: ISwButtonProps) {
+  const { pending } = props;
+  return (
+    <Button
+      {...props}
```
Comment (ulfryk) [inline src/input/button/button.tsx]: shuld be without `pending` (can use spread in props destructuring `{ className, pending, spinnerClassName,  …bsProps }`)

File: src/input/button/button.sass (hunk 2 above)
Comment (tomecko) [reply src/input/button/button.sass]: this is outside spinner
so maybe `margin` but not necessarily

**Pull request merged**